[2023-09-04T06:02:03.541+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-02-27T00:00:00+00:00 [queued]>
[2023-09-04T06:02:03.560+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-02-27T00:00:00+00:00 [queued]>
[2023-09-04T06:02:03.561+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:02:03.790+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2022-02-27 00:00:00+00:00
[2023-09-04T06:02:03.800+0000] {standard_task_runner.py:57} INFO - Started process 5122 to run task
[2023-09-04T06:02:03.805+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2022-02-27T00:00:00+00:00', '--job-id', '346', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp4mo3y5b4']
[2023-09-04T06:02:03.808+0000] {standard_task_runner.py:85} INFO - Job 346: Subtask delete_data_from_table
[2023-09-04T06:02:03.909+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-02-27T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:02:04.051+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2022-02-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-27T00:00:00+00:00'
[2023-09-04T06:02:04.053+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2022-02-27' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:02:04.066+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:04.079+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:04.094+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2022-02-27' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:02:04.100+0000] {sql.py:427} INFO - Rows affected: 0
[2023-09-04T06:02:04.117+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20220227T000000, start_date=20230904T060203, end_date=20230904T060204
[2023-09-04T06:02:04.178+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:02:04.213+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:04:45.422+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-02-27T00:00:00+00:00 [queued]>
[2023-09-04T06:04:45.462+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-02-27T00:00:00+00:00 [queued]>
[2023-09-04T06:04:45.463+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:04:45.530+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2022-02-27 00:00:00+00:00
[2023-09-04T06:04:45.543+0000] {standard_task_runner.py:57} INFO - Started process 6405 to run task
[2023-09-04T06:04:45.550+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2022-02-27T00:00:00+00:00', '--job-id', '886', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp2v3s5zhu']
[2023-09-04T06:04:45.555+0000] {standard_task_runner.py:85} INFO - Job 886: Subtask delete_data_from_table
[2023-09-04T06:04:45.756+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-02-27T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:04:46.103+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2022-02-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-27T00:00:00+00:00'
[2023-09-04T06:04:46.107+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2022-02-27' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:04:46.134+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:46.165+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:46.216+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2022-02-27' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:04:46.232+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:04:46.272+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20220227T000000, start_date=20230904T060445, end_date=20230904T060446
[2023-09-04T06:04:46.703+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:04:46.794+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
